packages/nocodb/src/lib/meta/api/utilApis.ts

```diff
@@ -23,6 +23,14 @@ export async function testConnection(req: Request, res: Response) {
 
 export async function appInfo(req: Request, res: Response) {
   const projectHasAdmin = !(await User.isFirst());
+  const oidcAuthName = !!(process.env.NC_OIDC_ISSUER &&
+    process.env.NC_OIDC_AUTH_URL &&
+    process.env.NC_OIDC_TOKEN_URL &&
+    process.env.NC_OIDC_USERINFO_URL &&
+    process.env.NC_OIDC_CLIENT_ID &&
+    process.env.NC_OIDC_CLIENT_SECRET) ?
+      ( process.env.NC_OIDC_DISPLAY_NAME ?? 'Your Identity Provider' ) :
+      '';
```

Comment:
Return `null` as auth name instead of empty string so we can enforce the checks as mentioned above.